lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "menu_item"
+require "csv"
+
+class Menu
+
+  DEFAULT_FILENAME = "./lib/takeaway_menu.csv"
+  attr_reader :m_items
+
+  def initialize
+    @m_items = {}
+  end
+
+  def load_from_csv
+    @m_items = {}
+    CSV.foreach(DEFAULT_FILENAME, headers: true) do |row|
```

Comment:
passing DEFAULT_FILENAME in here has made this method trickier to test than it needs to be.

I'd consider passing it into the method instead:
``` ruby
def load_from_csv(file = DEFAULT_FILENAME)
  #do some stuff
end
```